apps/api/src/domain/factories/__tests__/buildAggregateFactoryTestCases/index.ts

```diff
@@ -34,6 +35,7 @@ export type FactoryTestSuiteForAggregate<TAggregateType extends AggregateType =
 
 export default (): FactoryTestSuiteForAggregate[] => [
     buildBibliographicReferenceFactoryTestSet(),
+    buildDigitalTextFactoryTestSet(),
```

Comment:
Just want to check in about comments above in `digital-text.aggregate-factory.test-set.ts`

Reply:
Ah yea. It can stay for now. I realized after the fact that the dto-based factories are still used in create commands. But I'm going to have a closer look at this soon.

Reply:
Basically, we only need a sanity check at the level of the factory that the invariant validation errors come through, and then a happy path test. I think we want to check that null or undefined DTOs also raise appropriate errors, given that this never hits the invariant validation in the first place. I just want to note that we do test invariant validation comprehensively elsewhere.